string data
string changes
cobots_core/Version currentTag
cobots_core/Version previousTag
